features/prompt_memes_c/challenge_promptmeme_deletion.feature

```diff
@@ -187,7 +188,8 @@ Feature: Prompt Meme Challenge
   Then I should see "Collection was successfully deleted."
     And no one should have a claim in "Battle 12"
     And no one should be signed up for "Battle 12"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.